multiversx_sdk/abi/token_identifier_value.py

```diff
@@ -0,0 +1,13 @@
+
+
```

Comment:
unnecessary empty lines but we can live with it :)

Reply:
Fixed in https://github.com/multiversx/mx-sdk-py/pull/49.